model/resource/os.yaml

```diff
@@ -13,3 +13,16 @@ groups:
       - ref: os.name
       - ref: os.version
       - ref: os.build_id
+        note: >
+          Resource detector SHOULD fetch values from following sources:
```

Comment:
```suggestion
          `build_id` values SHOULD be obtained from the following sources:
```
I think we usually omit the component that is "fetching" the values. So I suggest something like this.